package.json

```diff
@@ -61,7 +61,8 @@
     "test": "react-scripts test",
     "test:coverage": "yarn test --coverage --watchAll=false",
     "eject": "react-scripts eject",
-    "reveal": "ts-node ./manualreveal"
+    "reveal": "ts-node ./manualreveal",
+    "parse-approved-identifiers": "ts-node ./src/common/helpers/proposalTitleAndDescription/approvedIdentifiersTableParser.ts"
```

Comment:
This script parses the approved identifiers in the approved identifiers table text file https://raw.githubusercontent.com/UMAprotocol/docs/master/docs/uma-tokenholders/approved-price-identifiers.md

We will automate this process for UMA 2.0

Reply:
this is awesome! great way to do this. can you add something to the docs around how to run this?

Reply:
Absolutely will do.